api/mapping/views.py

```diff
@@ -354,6 +355,14 @@ class SignUpView(generic.CreateView):
     template_name = 'registration/signup.html'
 
 
+class CCPasswordChangeView(PasswordChangeView):
```

Comment:
Why did we change the classes for password change in views.py and enter in a pass statement?